### PR TITLE
App icon change

### PR DESCRIPTION
**Pull request by jacsonah**

### CODE REVIEW

**Comment by jacsonah** [comment]
Sorry, but what tool did you use to measure these values? I'm a little confused about which part I should align.

**Comment by mpobaschnig** [comment]
I use [Inkscape](https://flathub.org/apps/details/org.inkscape.Inkscape) for that. Either via measurement tool or using rectangles to measure on the page grid. When turning on page grid, you can easily see symmetry or alignment issues, see these pictures, where the black rectangles are grid aligned:

Screenshot from 2021-10-30 18-07-27

Screenshot from 2021-10-30 18-12-08

Screenshot from 2021-10-30 18-14-06

As comparison, Password Safe icon alignment:

Screenshot from 2021-10-30 18-02-01

Basically every element should be aligned with the grid here, e.g. circles should align with top/left/bottom/right side, lines should align, and so on.

**Review by mpobaschnig — CHANGES_REQUESTED** [review]
The inner, bright part is now shifted by 2px to the bottom, hiding the "step"/darker shade.

Still missing:

- Inner circles are not grid aligned
- Metal bars are not grid aligned

**Review by mpobaschnig — CHANGES_REQUESTED** [review]
The inner, bright part is still shifted by 2px to the bottom, hiding the "step"/darker shade.

Moving this bright inner part and the cross/circles in the middle by 2px up, then it's ready.

**Comment by Crono23** [comment]
Hi, sorry if I sound impatient. I've been watching this merge request and it's almost a month since progress was made. Is it going to be finished or may I propose my own icon design?

**Comment by mpobaschnig** [comment]
There is not much left from the design side, so I don't think it's worth the time working on a new icon now.

@jacsonah If it's okay for you, I'll just fix the outstanding issues myself and merge it.

**Comment by jacsonah** [comment]
> There is not much left from the design side, so I don't think it's worth the time working on a new icon now.
> 
> @jacsonah If it's okay for you, I'll just fix the outstanding issues myself and merge it.

You can fix the outstanding issues, it's ok for me.

**Comment by mpobaschnig** [comment]
Cherry picked commits and pushed to master, since the diff between branches is too big. Thanks a lot!
